raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -59,6 +60,7 @@
 import static org.junit.jupiter.api.Assertions.fail;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
```

Comment:
Is ```assumeTrue``` used in this test? I test ```RaftEventSimulationTest`` but there is no ignored test cases.

Reply:
Good catch, removed since it's not currently needed.